auth-api/src/auth_api/services/org.py

```diff
@@ -708,7 +716,13 @@ def search_orgs(search: OrgSearch, environment):  # pylint: disable=too-many-loc
         }
         include_invitations: bool = False
         search.access_type, is_staff_admin = Org.refine_access_type(search.access_type)
-        if search.statuses and OrgStatus.PENDING_ACTIVATION.value in search.statuses:
+
+        if search.state == EFTShortnameState.UNLINKED.value:
+            eft_payment_accounts = Org.get_eft_payment_accounts()
```

Comment:
Looks like we are fetching all the eft accounts for every request. After the auth search we should have the ids we need to pass onto pay_api.